### ISSUE TITLE
[JOSE] Release versioning to match JOSE publication

### ISSUE THREAD

**Issue by darothen** [carpentries-lab/python-aos-lesson#14]
Submission to JOSE is indicated as v1.0.0, but the most recent release tag is v9.3.1. I think tweaks could be made in either place (here or in the JOSE submission); if a new v1.0.0 release is cut here, it would be helpful to add a note to the `README.md` indicating how the sequence of version numbers diverges.

CC: openjournals/jose-reviews#37

**Comment by DamienIrving** [comment]
Thanks for pointing out this inconsistency, @darothen. I haven't actually tagged any releases of the lessons - those previous releases relate to the development of the [Carpentries lesson template](https://github.com/carpentries/lesson-example) and were inherited when I used GitHub's importer to create my own copy.

I'm thinking that once I've addressed reviewer comments and the JOSE paper is accepted for publication, I'll create a new v1.0.0 tag (adding a note about disregarding earlier version numbers).
